odoo/addons/base/models/ir_cron.py

```diff
@@ -50,7 +50,7 @@ class ir_cron(models.Model):
     ir_actions_server_id = fields.Many2one(
         'ir.actions.server', 'Server action',
         delegate=True, ondelete='restrict', required=True)
-    cron_name = fields.Char('Name', related='ir_actions_server_id.name', store=True, readonly=False)
+    cron_name = fields.Char('Name', compute='_compute_cron_name', store=True)
```

Comment:
This field was editable.  Is it actually modified by some code, or put in some form to be edited?

Reply:
Oh I see.  Possibly `readonly=False` was not intentional, but accidental and kept by backward compatibility.